sigs/io/RELEASE.md

```diff
@@ -40,4 +40,5 @@ Please provide both GitHub and PyPI handle to join the release team.
 
 Current Release Team:
 - Yong Tang - GitHub: [@yongtang](https://github.com/yongtang) - PyPI: [yongtang](https://pypi.org/user/yongtang)
+- Anthony Dmitriev - GitHub: [@dmitrievanthony](https://github.com/dmitrievanthony) - PyPI: [mitrievanthony](https://pypi.org/user/dmitrievanthony)
```

Comment:
I guess it is a typo, should be `PyPI: [dmitrievanthony]`.

Reply:
Ah sorry, my typo. Corrected.